packages/cactus-plugin-ledger-connector-quorum/README.md

```diff
@@ -86,6 +86,125 @@ To check that all has been installed correctly and that the pugin has no errors,
 ```sh
 npm run test:plugin-ledger-connector-quorum
 ```
+
+### Building/running the container image locally
+
+In the Cactus project root say:
+
+```sh
+DOCKER_BUILDKIT=1 docker build -f ./packages/cactus-plugin-ledger-connector-quorum/Dockerfile . -t cplcb
+```
+
+Build with a specific version of the npm package:
+```sh
+DOCKER_BUILDKIT=1 docker build --build-arg NPM_PKG_VERSION=0.4.1 -f ./packages/cactus-plugin-ledger-connector-quorum/Dockerfile . -t cplcb
+```
+
+#### Running the container
+
+Launch container with plugin configuration as an **environment variable**:
+```sh
+docker run \
+  --rm \
+  --publish 3000:3000 \
+  --publish 4000:4000 \
+  --env PLUGINS='[{"packageName": "@hyperledger/cactus-plugin-ledger-connector-quorum", "type": "org.hyperledger.cactus.plugin_import_type.LOCAL",  "options": {"rpcApiHttpHost": "http://localhost:8545", "instanceId": "some-unique-quorum-connector-instance-id"}}]' \
```

Comment:
@jordigiam Is it not needed to add here the keychain memory plugin? I was just guessing that it is based on you having included the npm install of it in the image. What I'm trying to make sure is that these example commands (all of them) still work with the updates to the image you just made, if you can confirm that then I'm good.

Reply:
If I delete the keychain-memory from the dockerfile I can't install the keychain-memory with the docker run command, it failed with a error: "Cannot find module "uuid""

Reply:
Added dependency in the description. Approving now because once that issue is resolved the examples in the README here should start working automatically (once npm has the new code published on the default dist-tag)